CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ To contribute code changes or update the documentation, please follow these step
 5. Create a pull request to the main repository with an explanation of your changes. The PR should detail the
    contribution and link to any related issues - if existing.
 
-## Code contribution guidelines
+## Code Contribution Guidelines
```

Comment:
I'm guessing that pre-commit made these updates?

Reply:
I re-pascal-cased the heading as per a discussion in discord, but they can be reverted. 😄